src/cdk/a11y/key-manager/list-key-manager.ts

```diff
@@ -188,6 +188,12 @@ export class ListKeyManager<T extends ListKeyManagerOption> {
     return this;
   }
 
+  /** Cancels the current typeahead sequence. */
+  cancelTypeahead(): this {
+    this._pressedLetters = [];
```

Comment:
Just double-checking: From high-level glancing at the issue, would they rather expect the current buffer to be written immediately on blur?

Reply:
I think that it's open to interpretation. My take was that if focus leaves the trigger, we don't know which item the user was intending to focus.

Reply:
I understand, but I think that both behaviors can be equally valid. https://github.com/angular/components/issues/25452 talks about it being confusing that we can write the value after they've tabbed away.

Reply:
yeah, as said, this was mostly to check if you considered the other alternative. I'm good either way